app/views/employees/_form.html.erb

```diff
@@ -11,6 +11,11 @@
     </div>
   <% end %>
 
+  <div class="field">
+    <%= form.label :store %>
+    <%= form.collection_select(:store_id, Store.all, :id, :name)%>
```

Comment:
Personally, I think this line leaves the call to Store a bit exposed. I would recommend using [select_tag](https://apidock.com/rails/ActionView/Helpers/FormTagHelper/select_tag).

Reply:
"Store is exposed", can you please! expand on it, I search for info on the matter but it seems no issues to use collection.

Reply:
Views should contain as little logic as possible. It is important to make sure you never write code that allows for a security flaw.

Considering the above, you should call the model in a helper so as not to mess up the view so much (in which there should be the least possible logic).

Example:

In your helper file make the call to get the Store values:

```ruby
def stores_for_select
  Store.all.pluck(:id, :nombre)
end
```

And in the view, call the method created in your helper.

```suggestion
    <%= form.select :store_id, stores_for_select %>
```